Components/Pages/MovesGallery.razor

```diff
@@ -110,12 +110,9 @@ else
         }
     }
 
-    private async Task OpenDialogAsync()
+    private void NavigateToAdd()
```

Comment:
await jest w środku a tutaj void - wiem , że Blazor chyba sobie z tym poradzi - ale zweryfikuj to czy nie lepiej zmienić na Task - pewny nie jestem - do sprawdzenia przez Ciebie po prostu

Reply:
z tego co wiem jest typu void i w takiej postać może być umieszczany w metodach Task asynchronicznych